src/Stratis.Bitcoin.Features.BlockStore/BlockStoreSignaled.cs

```diff
@@ -226,9 +226,10 @@ protected override void OnNextCore(Block block)
             }
 
             // Announce the blocks to each of the peers.
-            IEnumerable<BlockStoreBehavior> behaviours = peers.Select(s => s.Behavior<BlockStoreBehavior>());
+            List<BlockStoreBehavior> behaviours = peers.Select(s => s.Behavior<BlockStoreBehavior>())
+                .Where(b => b != null).ToList();
```

Comment:
ToList is not needed here

Reply:
To List here is to prevent multiple enumeration : one with "Count", and one with foreach :)